pkg/agent/cluster.go

```diff
@@ -70,9 +71,34 @@ func NewCluster(ctx context.Context, assetDir, rendezvousIP, kubeconfigPath, ssh
 	czero := &Cluster{}
 	capi := &clientSet{}
 
-	authToken, err := FindAuthTokenFromAssetStore(assetDir)
-	if err != nil {
-		logrus.Fatal(err)
+	var authToken string
+	var err error
+
+	authToken, _ = FindAuthTokenFromAssetStore(assetDir) //nolint:errcheck
```

Comment:
It does not seem correct to ignore all the possible errors. IINW The only error condition that could be tolerated it's the `token not found` one, and only for the add-nodes workflow. So, better to let the ` FindAuthTokenFromAssetStore` return a proper error, and check it if it could be ignored or not.

Reply:
By design, for the add-nodes workflow the `FindAuthTokenFromAssetStore` will always fail, so it could probably make more sense to move it within the install workflow case only